tests/config_test.go

```diff
@@ -620,6 +860,34 @@ func withDownwardAPI(name string) libvmi.Option {
 	}
 }
 
+func withConfigMapFs(configMapName, volumeName string) libvmi.Option {
```

Comment:
Can we move these function into `tests/libvmi` in this way they could be reused?

Reply:
I'll do it